var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -79,4 +86,11 @@ def cmake_args(self):
             options.append('-DCUDA_TOOLKIT_ROOT_DIR:STRING=' +
                            self.spec['cuda'].prefix)
 
+        if self.spec.variants['simd'].value != 'AUTO':
```

Comment:
Can you exclude that block for gromacs-5.1?

Reply:
I do not see why. Some people may want to have specific SIMD instructions with 5.1, too.

Reply:
Do you think we can leave it like this? Would be cool if the PR could be merged at some point.

Reply:
Ok